travis/git_run.py

```diff
@@ -22,6 +22,8 @@ def run(self, command):
             res = subprocess.check_output(cmd)
         except subprocess.CalledProcessError:
             res = None
+        if isinstance(res, bytes):
+            res = res.decode()
```

Comment:
what about use all in bytes?

Using res.strip(b’\n’) and so on

Reply:
I think this change is less invasive because the method `git.run` is executed many times

Are you agree?